test/integration/hooks/validate.test.js

```diff
@@ -108,47 +107,6 @@ describe(Support.getTestDialectTeaser('Hooks'), function() {
         });
       });
     });
-
-    describe('on error', function() {
```

Comment:
Keep these tests as they are, we need at least a few test running in integration, just to be sure